src/ui/chat-interface.tsx

```
@@ -43,650 +43,632 @@ import { useModel } from "@/context/ModelContext";
+
+				setImages((prev) => [...prev, ...arraizedImages]);
+			} catch (error) {
+				console.error("Error handling paste:", error);
+				alert("Error handling paste. Please try again.");
+			}
+		},
+		[checkFileSize]
+	);
+
+	// Function to handle Drag and Drop
+	const handleDragAndDrop = useCallback(
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)